readme.md

```diff
@@ -26,18 +26,35 @@ Note: You do need CodeComplice for Sublime Text: https://github.com/spectacles/C
 
 ### Install
 
+#### For Laravel 5.5 or later
+
 Require this package with composer using the following command:
 
 ```bash
 composer require --dev barryvdh/laravel-ide-helper
 ```
 
+#### For Laravel 5.4 or earlier
+
+Laravel 5.4 or earlier versions do not have Package Auto-Discovery, so you need to add the ServiceProvider manually.
+
+Require this package with composer using the following command:
+
+```bash
+composer require barryvdh/laravel-ide-helper
```

Comment:
Why would we still recommend by default the **non** `--dev` version for <5.5?

Reply:
agreed